src/openrct2/title/TitleScreen.cpp

```diff
@@ -187,7 +191,7 @@ void TitleScreen::TitleInitialise()
         IScenarioRepository * scenarioRepository = GetScenarioRepository();
         _sequencePlayer = CreateTitleSequencePlayer(scenarioRepository);
     }
-    size_t seqId = title_sequence_manager_get_index_for_config_id(gConfigInterface.current_title_sequence_preset);
+    size_t seqId = title_get_config_sequence();
     if (seqId == SIZE_MAX)
```

Comment:
The new function returns `uint16` instead of `size_t`, which renders this check invalid.

@trigger-death can you please fix that?